demos/python_demos/clip_image_classification/README.md

```diff
@@ -80,3 +80,34 @@ time percentile 50: 90.00 ms; speed percentile 50: 11.11 fps
 time standard deviation: 0.00
 time variance: 0.00
 ```
+
+## Requesting detection name with rest request
+
+Run the rest client script
+```bash
+python3 rest_client.py --http_port 8000
```

Comment:
why not `--url localhost:8000` to match grpc client?

Reply:
I have done it on purpose - this is a convention we use in all other rest api demos. If the user has to change the script parameters he will not confuse the API endpoint he is using ( rest/grpc )

Reply:
Either switch to url or to rest_port.